CONTRIBUTING.md

```diff
@@ -0,0 +1,82 @@
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+## Open Development & Community Driven
+
+This project is open-source under the [MIT license](LICENSE). All the work done is available on GitHub.
+
+## Code of Conduct
+
+This project and everyone participating in it are governed by a [Code of Conduct](CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please read the [full text](CODE_OF_CONDUCT.md) so that you can read which actions may or may not be tolerated.
+
+## Bugs
+
+We are using [GitHub Issues](https://github.com/Lith/strapi-provider-upload-google-cloud-storage/issues) to manage our public bugs. We keep a close eye on this so before filing a new issue, try to make sure the problem does not already exist.
+
+---
+
+## Pull Request Process
+
+The team will review your pull request and will either merge it, request changes to it, or close it.
+
+**Before submitting your pull request** make sure the following requirements are fulfilled:
+
+1. Fork the repository and create your branch from `master`.
+    - Run `yarn install` in the repository root.
+    - If you’ve fixed a bug or added code that should be tested, add the tests and then link the corresponding issue in either your commit or your PR!
+    - Ensure the test suites are passing:
+      - `yarn test`
+    - Make sure your code lints 
+      - `yarn lint`
+2. Update the [README.md](README.md) with details of changes to the interface, this includes new environment 
+   variables, exposed ports, useful file locations and container parameters.
+3. You need to Signed-off-by all of your commits before push, like this example :
+    ```shell script
+    This is my commit message
+    
+    Signed-off-by: Random J Developer <random@developer.example.org>
+    ```
+   Git even has a -s command line option to append this automatically to your commit message:
+   ```
+   $ git commit -s -m 'This is my commit message'
+   ```
+4. A team leader will check, suggest modification, approve, merge, and close your request.
+
+## Contribution Prerequisites
+
+* You have [Node](https://nodejs.org/en/) at v10.x.x only and [Yarn](https://yarnpkg.com/en/) at v1.2.0+.
+* You are familiar with Git.
+
+---
+
+## Miscellaneous
+
+### Repository Organization
+
+We chose to to use a monorepo design that exploits [Yarn Workspaces](https://yarnpkg.com/en/docs/workspaces) in the way [React](https://github.com/facebook/react/tree/master/packages) or [Babel](https://github.com/babel/babel/tree/master/packages) does. This allows the community to easily maintain the whole ecosystem, keep it up-to-date and consistent.
+
+We do our best to keep the master branch as clean as possible, with tests passing at all times. However, it may happen that the master branch moves faster than the release cycle. Therefore check the [releases on npm](https://www.npmjs.com/package/strapi) so that you're always up-to-date with the latest stable version.
```

Comment:
`npm` is linking to strapi, is it right?

Reply:
🤔  I think I need to change this part, I've get this from Strapi contributing guide, I will remove this part.